example.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
I think this got coppied here on accident? Should be removed. If that makes the tests fail, then we have more work to do.